Source/ExtensionMethods/RectangleFExtensions.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Drawing.Drawing2D;
+using System.Text;
+
+namespace Svg.ExtensionMethods
+{
+    internal static class RectangleFExtensions
+    {
+        internal static Rectangle Round(this RectangleF rectangle)
```

Comment:
Why not use [Rectangle.Round](https://docs.microsoft.com/dotnet/api/system.drawing.rectangle.round) ?

Reply:
Good point, I wasn't aware it existed.